File: Backend/Models/Film.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace Models
{
    public class Film
    {
        [Key]
        public int ID { get; set; }
        [Required]
        [MaxLength(50)]
        public string Naslov { get; set; }
        [Required]
        [MaxLength(50)]
        public string Tip { get; set; }
        [Required]
        [Range(1,10)]
        public int Rejting { get; set; }
        [Required]
        [Range(1930,2021)]
        public int Godina { get; set; }
        [MaxLength(100)]
        public string Nominacija_za_nagrade { get; set; }
        [MaxLength(100)]
        public string Dobijene_nagrade { get; set; }
        [JsonIgnore]
        public Diskovi Diskovi { get; set; }
        
        public List<Reziseri> Reziseri { get; set; }
        
        public List<Glumci> Glumci { get; set; }
    }
}

File: Klijent/Film.js
export class Film{
    constructor(naslov,tip,rejting,godina,nominacija_za_nagrade,dobijene_nagrade){
        this.naslov=naslov;
        this.tip=tip;
        this.rejting=rejting;
        this.godina=godina;
        this.nominacija_za_nagrade=nominacija_za_nagrade;
        this.dobijene_nagrade=dobijene_nagrade;
        this.kontejner=null;
    }
    crtajFilm(host){
        var tr= document.createElement("tr");
        host.appendChild(tr);

        var el=document.createElement("td");
        el.innerHTML=this.naslov;
        tr.appendChild(el);
        el=document.createElement("td");
        el.innerHTML=this.tip;
        tr.appendChild(el);
        el=document.createElement("td");
        el.innerHTML=this.rejting;
        tr.appendChild(el);
        el=document.createElement("td");
        el.innerHTML=this.godina;
        tr.appendChild(el);
        el=document.createElement("td");
        el.innerHTML=this.nominacija_za_nagrade;
        tr.appendChild(el);
        el=document.createElement("td");
        el.innerHTML=this.dobijene_nagrade;
        tr.appendChild(el);
    }
}